packages/docusaurus-theme-classic/src/index.js

```diff
@@ -22,6 +22,16 @@ class DocusaurusThemeDefault {
   getThemePath() {
     return path.resolve(__dirname, './theme');
   }
+
+  configureWebpack() {
+    return {
+      resolve: {
+        alias: {
+          '@theme/SearchAlgolia': path.resolve(__dirname, './noopModule.js'),
```

Comment:
Wondering if we should not name it SearchAlgolia -> but rather a more generic name. Maybe just `SearchBar` or `SearchBox` ?. Theme-classic is now also not tied to SearchAlgolia.

If there is a user who wanted to use another theme-search-xxx and theme-classic, they have to name the component as SearchAlgolia ?